source/server/well_known_names.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+
+#include "common/singleton/const_singleton.h"
+
+namespace Envoy {
+namespace Server {
+
+/**
+ * Well-known active UDP listener names.
+ */
+class UdpListenerNameValues {
+public:
+  const std::string RawUdp = "raw_udp_listener";
+  const std::string Quic = "quic_listener";
```

Comment:
I think this will be defined by the extension itself, right? I don't think we need this here.

Reply:
removed. I think it should be added here later instead of under extension/